lib/commands/performance.js

```diff
@@ -0,0 +1,339 @@
+              
+          let returnIndex = 1;
+          for (let i=0 ; i < arrayList.length ; i++){
+
+            data = arrayList[i];
+            start = data.indexOf(netVariables[index][0]);
+
+            if (start >= 0 ){
+
+              delimiter = data.indexOf("=", start+1);
+              end = data.indexOf(" ", delimiter+1);
+              bucketStart = data.substring(delimiter+1, end).trim();     
+
+              if (end > delimiter){
+                start = data.indexOf(netVariables[index][1], end+1);
+                if (start >= 0){
+                  delimiter = data.indexOf("=", start+1);
+                  end = data.indexOf(" ", delimiter+1);
+                  activeTime = data.substring(delimiter+1, end).trim();              
+                }
+              }
+                    
+              if (end > delimiter){
+                start = data.indexOf(netVariables[index][2], end+1);
+                if (start >= 0){
+                  delimiter = data.indexOf("=", start+1);
+                  end = data.indexOf(" ", delimiter+1);
+                  rxBytes = data.substring(delimiter+1, end).trim(); 
+                }             
+              }
+                    
+              if (end > delimiter){
+                start = data.indexOf(netVariables[index][3], end+1);
+                if (start >= 0){
+                  delimiter = data.indexOf("=", start+1);
+                  end = data.indexOf(" ", delimiter+1);
+                  rxPackets = data.substring(delimiter+1, end).trim(); 
+                }             
+              }
+                    
+              if (end > delimiter){
+                start = data.indexOf(netVariables[index][4], end+1);
+                if (start >= 0){
+                  delimiter = data.indexOf("=", start+1);
+                  end = data.indexOf(" ", delimiter+1);
+                  txBytes = data.substring(delimiter+1, end).trim();   
+                }           
+              }
+                    
+              if (end > delimiter){
+                start = data.indexOf(netVariables[index][5], end+1);
+                if (start >= 0){
+                  delimiter = data.indexOf("=", start+1);
+                  end = data.indexOf(" ", delimiter+1);
+                  txPackets = data.substring(delimiter+1, end).trim(); 
+                }             
+              }
+
+              if (end > delimiter){
+                start = data.indexOf(netVariables[index][6], end+1);
+                if (start >= 0){
+                  delimiter = data.indexOf("=", start+1);
+                  end = data.length;
+                  operations = data.substring(delimiter+1, end).trim(); 
+                        
+                }             
+              }
+              returnValue[returnIndex ++] = [bucketStart, bucketDuration, activeTime, rxBytes, rxPackets, txBytes, txPackets, operations];      
+            }
+          }
+        }
+        
+      }
```

Comment:
this method has lots of duplication and would benefit from being rewritten in a more functional style using methods from libraries like lodash.